### ISSUE TITLE
commandline and python api parsed same sql different output?

### ISSUE THREAD

**Issue by wonb168** [reata/sqllineage#247]
mysql:`
create table e as
with a as(select * from bb b)
, c as( select * from a)
select c.id, d.name 
from c 
inner join dd d on c.id=d.id`

sqllineage -g -f demo.sql
output column linage is right:
bb.* -> a.* -> c.*
c.id -> e.id
dd.name -> e.name

but, in python```
from sqllineage.runner import LineageRunner
sql="""create table e as
with a as(select * from bb b)
, c as( select * from a)
select c.id, d.name 
from c 
inner join dd d on c.id=d.id"""
result = LineageRunner(sql)
result.print_column_lineage()```
output:
<default>.e.id <- c.id
<default>.e.name <- <default>.dd.name
lost some lineage?

**Comment by reata** [comment]
Yes, it was designed this way and from your use case I think this was a bad design.

Previous design was that the command line interface will only give you back column lineage that ends with column in a table. If it ends in a subquery, that we won't show it. Meanwhile, the UI always shows everything. The reason I want to hide those lineage ending in subquery is that those columns are not used.

And c.* -> c.id, we should be able to do the mapping here.

Anyway, this is a good example. Let me make the command line interface and UI consistent first.
